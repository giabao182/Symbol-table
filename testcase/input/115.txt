INSERT a number
BEGIN
BEGIN
BEGIN
INSERT a string
LOOKUP a
END
LOOKUP a
END
LOOKUP a
END