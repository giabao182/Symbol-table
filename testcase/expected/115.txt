success
success
3
0
0